crates/bevy_diagnostic/Cargo.toml

```diff
@@ -21,13 +21,12 @@ bevy_log = { path = "../bevy_log", version = "0.12.0-dev" }
 bevy_time = { path = "../bevy_time", version = "0.12.0-dev" }
 bevy_utils = { path = "../bevy_utils", version = "0.12.0-dev" }
 
-# MacOS
-[target.'cfg(all(target_os="macos"))'.dependencies]
+# iOS
+[target.'cfg(all(target_os="ios"))'.dependencies]
```

Comment:
isn't this also needed for macOS? from sysinfo docs it seems it is https://docs.rs/sysinfo/latest/sysinfo/#use-in-binaries-running-inside-the-macos-or-ios-sandboxstores

Reply:
Only if running it in a VM. Talk about a nightmare...